src/CDSAPI.jl

```diff
@@ -1,88 +1,108 @@
 module CDSAPI
 
+## Base Modules Used
+using Base64
+using Dates
+using Logging
+using Printf
+
+## Modules Used
 using HTTP
 using JSON
-using Base64
 
 export
-    py2ju
+        retrieve, cdskeys
 
 """
-    retrieve(name, params, filename)
-
-Retrieves data for `name` from the Climate Data Store
-with the specified `params` and stores it in the current
-directory as `filename`.
+    retrieve(
+        dataset::AbstractString,
+        params::AbstractDict,
+        fname::AbstractString,
+        keys::AbstractDict = cdskeys()
+    )
+
+Retrieves datasets from the Climate Data Store, with options specified in a Julia Dictionary and saves it into a specified file.
+
+Arguments:
+    * `fname::AbstractString` : string that contains the path and name of the file that the data is to be saved into
+    * `dataset::AbstractString` : string specifies the name of the dataset within the Climate Data Store that the `retrieve` function is attempting to retrieve data from
+    * `params::AbstractDict` : dictionary that contains the keywords that specify the properties (e.g. date, resolution, grid) of the data being retrieved
+    * `keys::AbstractDict` : dictionary that contains API Key information read from the .cdsapirc file in the home directory (optional)
```

Comment:
Thanks for reordering the arguments. Maybe it would make sense to reorder the docstring as well?

Reply:
Done!